src/OpenSage.Game/Logic/Object/GameObject.cs

```diff
@@ -535,6 +602,10 @@ void AddBehavior(string tag, BehaviorModule behavior)
                 {
                     _body = body;
                 }
+                else if (module is IContainModule contain)
```

Comment:
this statement is starting to seem like a good candidate for a switch expression

Reply:
Done (assuming you meant switch statement).